### ISSUE TITLE
dyld: Library not loaded @rpath ERROR

### ISSUE THREAD

**Issue by kacz-nemo** [Kameleoon/client-swift#1]
Having updated kameleoonClient from '3.1' to '~> 4.0' I do get the following (crashing) error when running on the device: 

dyld[841]: Library not loaded: @rpath/kameleoonClient.framework/kameleoonClient
  Referenced from: <79AEB80F-F37F-378C-BBD7-549E607376E5> /private/var/containers/Bundle/Application/3973BC9E-858A-4F55-BF5B-FFEF10AD091B/TestKameleon.app/TestKameleon
  Reason: tried: '/usr/lib/swift/kameleoonClient.framework/kameleoonClient' (no such file, not in dyld cache), '/private/preboot/Cryptexes/OS/usr/lib/swift/kameleoonClient.framework/kameleoonClient' (no such file), '/private/var/containers/Bundle/Application/3973BC9E-858A-4F55-BF5B-FFEF10AD091B/TestKameleon.app/Frameworks/kameleoonClient.framework/kameleoonClient' (no such file), '/usr/lib/swift/kameleoonClient.framework/kameleoonClient' (no such file, not in dyld cache), '/private/preboot/Cryptexes/OS/usr/lib/swift/kameleoonClient.framework/kameleoonClient' (no such file), '/private/var/containers/Bundle/Application/3973BC9E-858A-4F55-BF5B-FFEF10AD091B/TestKameleon.app/Frameworks/kameleoonClient.framework/kameleoonClient' (no such file)
  
I do integrate via Cocoapods and on Simulator it runs smoothly... **Any idea on what the possible quick fix could be?**  In long run I wish to integrate Kameleoon into a  shared framework which would get integrated into the app, but here (above) I am sharing the log form a simple client app (TestKameleon.app)

```
  # Podfile
  platform :ios, '15.0'

  target 'TestKameleon' do
    use_frameworks!

    # Pods for TestKameleon
    pod 'kameleoonClient'
  end
```

**Comment by DmitryAE** [comment]
Dear @kacz-nemo,

We sincerely apologize for the delay in our response and regret any inconvenience this may have caused you. We are pleased to inform you that the issue you encountered has been effectively addressed and resolved in [Swift SDK version 4.1.4](https://github.com/Kameleoon/client-swift/blob/main/CHANGELOG.md#414---2024-04-17). While it is possible that the resolution occurred earlier, we highly recommend utilizing the most recent version of the SDK for optimal performance and reliability. You can easily install this version through Swift Package or Cocoapods.

If you require any further assistance or have additional inquiries, please do not hesitate to reach out to us.

Best regards,
Dmitry

**Comment by kacz-nemo** [comment]
👌 Many thanks Guys! I did find the issue no longer appearing in version 4.1.3. 

Best,
K
